lib/nitra/workers/rspec.rb

```diff
@@ -139,6 +139,19 @@ def runner_tags(runner)
       (runner.world.all_examples.map(&:metadata).flat_map(&:keys) - STANDARD_METADATA_KEYS).map(&:to_s)
     end
 
+    def filename_without_extension(filename)
+      # we could be running for a particular scenario (with bracketed location or linenumber)
+      if filename.include?('[')
+        line_number = filename.partition('[').last.partition(']').first.gsub(':', '_')
```

Comment:
Possibly a little simpler

```suggestion
        line_number = /\[(\d+:\d+)\]/.match(filename)[1].gsub(':', '_')
```

Reply:
The sequence can repeat:
`[1:2:3:2:1]` is valid too

Reply:
Maybe
` /\[(\d+(?::\d+)*)\]/.match(filename)[1].gsub(':', '_')`

Reply:
I guess if there are multiple `:`, this would not work tho. I'm having a further play with the regex.